roles/benchmarking_deploy_coco_dataset/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - role: check_deps
```

Comment:
Github helps you to give you a little nudge here.  Most files with no EOL are a no-no in my view.